docs/manage/fleet/_index.md

```diff
@@ -78,23 +78,20 @@ You can also revert to an earlier configuration from the History tab.
 For some configuration aspects you may require physical access to the robot so you can see how components are connected.
 {{< /alert >}}
 
-### Package Deployment
-
-_Coming soon._
-
-You and your collaborators can deploy control logic, [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](../../services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
-
 ### Remote control
 
-Everyone who has access to the robot can remotely control it using the [**Control** tab](robots/#control).
-This allows you to visually test and remotely operate robot components and services
+Everyone who has access to the robot can remotely control it on the app's [**Control** tab](robots/#control).
+This allows you to visually test and remotely operate robot components and services.
 
-### Control with code
+### Control with the SDKs
 
-Everyone who has access to the robot's location can obtain the robot address and secret needed to send API calls to the robot.
-You can also share the robot address and location secret without granting location access in the Viam app.
+Everyone who has access to the robot's location can obtain the robot's remote address and location secret needed to send API calls to the robot from the app's **Code sample** tab.
```

Comment:
```suggestion
Everyone who has access to the robot's location can obtain the robot's remote address and location secret from the app's **Code sample** tab, which are both needed to send API calls to the robot from the [Viam SDKs](/program/sdks/).
```

This is a good place to also link to the SDKs resource. The API calls emanate from the SDK/user app, not from the code sample tab ... but the address and secret can indeed be found on that tab. I've tried to clarify that in my suggestion above -- LMK what you think!